### PR TITLE
[bitnami/mxnet] Support for customizing standard labels

### PR DESCRIPTION
**Pull request by juan131**
### Description of the change

This PR follows up #18154 ensuring this chart is adapted to use new helpers' format, hence adding support for customizing standard labels.

### Benefits

User can customize standard labels.

### Possible drawbacks

None

### Applicable issues

N/A

### Additional information

#18154 must be merged **before** this PR

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/mxnet/templates/distributed/worker/statefulset.yaml [hunk 1]
```diff
@@ -249,15 +244,12 @@ spec:
...
-          {{- if .Values.commonAnnotations }}
-          {{- include "common.tplvalues.render" (dict "value" .Values.commonAnnotations "context" $) | nindent 10 }}
-          {{- end }}
+        {{- if or .Values.persistence.annotations .Values.commonAnnotations }}
+        {{- $claimAnnotations := merge .Values.persistence.annotations .Values.commonAnnotations }}
+        annotations: {{- include "common.tplvalues.render" ( dict "value" $claimAnnotations "context" $) | nindent 4 }}
+        {{- end }}
         {{- if .Values.commonLabels }}
-        labels: {{- include "common.tplvalues.render" (dict "value" .Values.commonLabels "context" $) | nindent 10 }}
+        labels: {{- include "common.labels.matchLabels" ( dict "customLabels"  .Values.commonLabels "context" $ ) | nindent 10 }}
```
Comment (FraPazGal) [inline bitnami/mxnet/templates/distributed/worker/statefulset.yaml:252]:
```suggestion
        labels: {{- include "common.labels.matchLabels" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 10 }}
```

**Review by FraPazGal — COMMENTED** [review]
Good PR @juan131! I added a couple of minor style comments.

File: bitnami/mxnet/templates/distributed/worker/statefulset.yaml (hunk 1 above)
Comment (juan131) [reply bitnami/mxnet/templates/distributed/worker/statefulset.yaml]: addressed at https://github.com/bitnami/charts/pull/18355/commits/99d48803e46b1696493ecc48b47c0a0ac3dfa029

File: bitnami/mxnet/templates/distributed/server/statefulset.yaml [hunk 2]
```diff
@@ -251,15 +246,12 @@ spec:
...
-          {{- if .Values.commonAnnotations }}
-          {{- include "common.tplvalues.render" (dict "value" .Values.commonAnnotations "context" $) | nindent 10 }}
-          {{- end }}
+        {{- if or .Values.persistence.annotations .Values.commonAnnotations }}
+        {{- $claimAnnotations := merge .Values.persistence.annotations .Values.commonAnnotations }}
+        annotations: {{- include "common.tplvalues.render" ( dict "value" $claimAnnotations "context" $) | nindent 4 }}
+        {{- end }}
         {{- if .Values.commonLabels }}
-        labels: {{- include "common.tplvalues.render" (dict "value" .Values.commonLabels "context" $) | nindent 10 }}
+        labels: {{- include "common.labels.matchLabels" ( dict "customLabels"  .Values.commonLabels "context" $ ) | nindent 10 }}
```
[the remark this answers is not in the document]
Comment (juan131) [reply bitnami/mxnet/templates/distributed/server/statefulset.yaml]: addressed at https://github.com/bitnami/charts/pull/18355/commits/99d48803e46b1696493ecc48b47c0a0ac3dfa029

**Review by FraPazGal — APPROVED** [review]

**Pull request merged**